--- httping-1.0.9/Makefile.orig	2006-01-01 19:18:58.000000000 +0100
+++ httping-1.0.9/Makefile	2006-01-02 15:28:04.000000000 +0100
@@ -14,9 +14,9 @@
 
 VERSION=1.0.9
 
-DEBUG=-g
 LDFLAGS+=-lssl -lcrypto $(DEBUG)
-CFLAGS+=-O2 -Wall -DVERSION=\"$(VERSION)\" $(DEBUG)
+CFLAGS ?= -O2
+override CFLAGS += -Wall -DVERSION=\"$(VERSION)\"
 
 OBJS=http.o io.o str.o error.o utils.o main.o tcp.o mssl.o res.o
 
