diff -uNr httping-2.2.old/main.c httping-2.2/main.c
--- httping-2.2.old/main.c	2013-04-04 21:46:48.000000000 +0200
+++ httping-2.2/main.c	2013-04-09 21:12:02.313689100 +0200
@@ -22,7 +22,7 @@
 #include <time.h>
 #include <sys/time.h>
 #ifdef NC
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 #endif
 
 #include "gen.h"
diff -uNr httping-2.2.old/nc.c httping-2.2/nc.c
--- httping-2.2.old/nc.c	2013-04-04 21:46:48.000000000 +0200
+++ httping-2.2/nc.c	2013-04-09 21:12:14.790931078 +0200
@@ -6,7 +6,7 @@
 #include <string.h>
 #include <sys/ioctl.h>
 
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 
 #include "error.h"
 #include "gen.h"
diff -uNr httping-2.2.old/nc.h httping-2.2/nc.h
--- httping-2.2.old/nc.h	2013-04-04 21:46:48.000000000 +0200
+++ httping-2.2/nc.h	2013-04-09 21:12:23.691341465 +0200
@@ -1,4 +1,4 @@
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 
 extern char win_resize;
 
